impl/sql/adminstorage/admin_storage.go

```diff
@@ -38,7 +38,7 @@ CREATE TABLE IF NOT EXISTS Domains(
   VRFPrivateKey         MEDIUMBLOB NOT NULL,
   MinInterval           BIGINT NOT NULL,
   MaxInterval           BIGINT NOT NULL,
-  Deleted               BOOLEAN,
+  Deleted               INTEGER,
```

Comment:
what motivates this?  has Deleted become a multi-state enum rather than a boolean?

Reply:
Oops just read the PR desc, I see it's a SQLite compatibility thing.  Ignore.

Reply:
💯